app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -38,6 +43,7 @@
     private static final String HABIT_TITLE_ERROR_MESSAGE = "Incorrect habit title entered";
     private static final String HABIT_REASON_ERROR_MESSAGE = "Incorrect habit reason entered";
     private static final String EMPTY_DATE_ERROR_MESSAGE = "Please enter a start date";
+    private static final String NO_DAYS_SELECTED_ERROR_MESSAGE = "Please select what days of the week.";
```

Comment:
This sentence seems incomplete, try saying it out loud. 

"Please select what days of the week."

This can probably be something like:
"Please select a weekly schedule" or something similar

Reply:
Resolved, I used your suggested sentence.